components/automate-chef-io/content/docs/iam-v2-guide.md

```diff
@@ -241,7 +241,7 @@ so they are not relevant when assigning IAM resources such as teams or roles.
 
 #### Configuring Project Limit
 
-By default, Chef Automate limits you to six projects. You can increase the project limit using the command line.
+By default, Chef Automate is currently limited to 300 projects. If you would like to increase that limit, you will need to do so using the Chef Automate CLI:
```

Comment:
So this line was originally reworked to decrease the use of passive tense and simplify the sentence structure. I recommend just changing the number to `300`.
```suggestion
By default, Chef Automate limits you to 300 projects. You can increase the project limit using the command line.
```

Reply:
I strictly swapped the numbers-- not sure how this old text took over. Maybe a bad rebase 🤷‍♀

Reply:
Totally understand. I was mildly suspicious the text change was a computer conspiracy, not actually by a user. It's so frustrating when that happens!